charts/sentry/templates/configmap-sentry.yaml

```diff
@@ -0,0 +1,443 @@
+{{- $redisPort := include "sentry.redis.port" . -}}
+{{- $redisPass := include "sentry.redis.password" . -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "sentry.fullname" . }}-sentry
+  annotations:
+    helm.sh/hook: pre-install,pre-upgrade
+    {{- if .Values.vault.vaultaddr }}
+    vault.security.banzaicloud.io/vault-addr: {{ .Values.vault.vaultAddr | quote }}
+    vault.security.banzaicloud.io/vault-role: {{ .Values.vault.vaultRole | quote }}
+    vault.security.banzaicloud.io/vault-path: {{ .Values.vault.vaultPath | quote }}
+    vault.security.banzaicloud.io/inline-mutation:  {{ .Values.vault.inlineMutation | quote }}
+    {{- end }}
+  labels:
+    app: sentry
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  config.yml: |-
+    {{- if .Values.system.adminEmail }}
+    system.admin-email: {{ .Values.system.adminEmail | quote }}
+    {{- end }}
+    system.secret-key: {{ .Values.system.secretKey | default (randAlphaNum 50) | quote }}
+    {{- if .Values.system.url }}
+    system.url-prefix: {{ .Values.system.url | quote }}
+    {{- end }}
+
+    # This URL will be used to tell Symbolicator where to obtain the Sentry source.
+    # See https://getsentry.github.io/symbolicator/api/
+    system.internal-url-prefix: 'http://{{ template "sentry.fullname" . }}-web:{{ .Values.service.externalPort }}'
+    symbolicator.enabled: {{ .Values.symbolicator.enabled }}
+    {{- if .Values.symbolicator.enabled }}
+    symbolicator.options:
+      url: "http://{{ template "sentry.fullname" . }}-symbolicator:{{ template "symbolicator.port" }}"
+    {{- end }}
+
+    ##########
+    # Github #
+    ##########
+    github-app.id: github-app.id: ${vault:secret/data/kubernetes/core/shared/main/sentry#github_appId}
+    github-app.name: ${vault:secret/data/kubernetes/core/shared/main/sentry#github_app_name}
+    github-app.private-key: |-
+    ${vault:secret/data/kubernetes/core/shared/main/sentry#github_privateKey}
+    github-app.webhook-secret: ${vault:secret/data/kubernetes/core/shared/main/sentry#github_webhook_secret}
+    github-app.client-id: ${vault:secret/data/kubernetes/core/shared/main/sentry#github_clientId}
+    github-app.client-secret: ${vault:secret/data/kubernetes/core/shared/main/sentry#github_client_secret}
+
+
+    ##########
+    # Google #
+    ##########
+    {{- if .Values.google.clientId }}
+    auth-google.client-id: {{ .Values.google.clientId | quote }}
+    auth-google.client-secret: {{ .Values.google.clientSecret | quote }}
+    {{ end }}
+
+    #########
+    # Slack #
+    #########
+    slack.client-id: ${vault:secret/data/kubernetes/core/shared/main/sentry#slack_clientId}
+    slack.client-secret: ${vault:secret/data/kubernetes/core/shared/main/sentry#slack_client_secret}
+    slack.signing-secret: ${vault:secret/data/kubernetes/core/shared/main/sentry#slack_signing_secret}
+
```

Comment:
Nos testes que fiz um dos pods sentry-worker esta apresentando esse error:
`!! Configuration error: ConfigurationError('Malformed config.yml file: mapping values are not allowed in this context\n  in "/etc/sentry/config.yml", line 11, column 29',)`